README.md

```diff
@@ -1,13 +1,30 @@
-# symphony-api-client-python
+[![License](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
```

Comment:
There are dynamic badges which retrieves information from the latest pypi release. But I didn't find any fetching the latest pre-release or a specific release in Pypi. Hence the static badges.